samples/StaticHtmlComposites/webpack.config.js

```diff
@@ -15,5 +15,28 @@ module.exports = {
     path: path.resolve(__dirname, 'dist'),
     libraryTarget: 'umd',
     library: '[name]'
+  },
+  devServer: {
+    port: 3000,
+    injectClient: false,
+    open: true,
+    proxy: [
+      {
+        path: '/token',
```

Comment:
when we switch to azure functions we need to make sure all our api server routes are `/api/*` to make these kind of things much simpler

Reply:
Thanks for this reminder, will keep this in my note